api/workspace.proto

```diff
@@ -57,6 +63,12 @@ message Workspace {
 	WorkspaceTemplate workspaceTemplate = 6;
 	WorkspaceStatus status = 7;
 	repeated KeyValue labels = 8;
+
+	string phase = 9;
```

Comment:
Is there a reason these also here? They’re in `WorkspaceStatus`. If this is to simplify, then we should remove `WorkspaceStatus` as a field.

Reply:
Removed it, having it in status is good.